[IMP] finance: avatax version 17 updates (#9341)
odoo/documentation: content/applications/finance/accounting/taxes/avatax.rst

   * - :guilabel:`Avatax`
     - `account_avatax`
     - Default *AvaTax* module. This module adds the base *AvaTax* features for tax calculation.
   * - :guilabel:`Avatax for geo localization`
     - `account_avatax_geolocalize`

>      - This module includes the features required for integration of AvaTax into geo-localization in

Reviewer: ```suggestion
     - This module includes the features required for integration of *AvaTax* into geo-localization in
```

       Odoo.
   * - :guilabel:`Avatax for SO`
     - `account_avatax_sale`
     - Includes the information needed for tax calculation on sales orders in Odoo.
   * - :guilabel:`Avatax for Inventory`
     - `account_avatax_stock`

>      - Includes tax calculation on Odoo Inventory.

Reviewer: ```suggestion
     - Includes tax calculation in Odoo *Inventory*.
```

   * - :guilabel:`Amazon/Avatax Bridge`
     - `sale_amazon_avatax`

>      - Includes tax calculation features between the Amazon connector and Odoo.

Reviewer: ```suggestion
     - Includes tax calculation features between the *Amazon Connector* and Odoo.
```
this suggestion is based on how you format the Amazon Connector below

   * - :guilabel:`Avatax Brazil`
     - `l10n_br_avatax`
     - Includes information for tax calculation in the Brazil localization.
   * - :guilabel:`Avatax Brazil for Services`
     - `l10n_br_avatax_services`

>      - This module includes the features required for tax calculation for services in the Brazil
>        localization.
>    * - :guilabel:`Avatax Brazil Sale for Services`
>      - `l10n_br_edi_sale_services`
>      - This module includes the features required for tax calculation for the sale of services in
>        the Brazil localization.
>    * - :guilabel:`Test SOs for the Brazilian AvaTax`
>      - `l10n_br_test_avatax_sale`
>      - This module includes the features required for test sales orders in the Brazil localization.

Reviewer: ```suggestion
     - This module includes the required features for tax calculation for services in the Brazil
       localization.
   * - :guilabel:`Avatax Brazil Sale for Services`
     - `l10n_br_edi_sale_services`
     - This module includes the required features for tax calculation for the sale of services in
       the Brazil localization.
   * - :guilabel:`Test SOs for the Brazilian AvaTax`
     - `l10n_br_test_avatax_sale`
     - This module includes the required features for test sales orders in the Brazil localization.
```
just an optional way to slightly enhance the "flow" of the explanations

> Click the :guilabel:`Install` button on the module labeled :guilabel:`Avatax`: `account_avatax`.

Reviewer: ```suggestion
Click the :guilabel:`Install` button on the module labeled, :guilabel:`Avatax`: `account_avatax`.
```

Doing so installs the following modules:

- :guilabel:`Avatax`: `account_avatax`
- :guilabel:`Avatax for SO`: `account_avatax_sale`
- :guilabel:`Avatax for Inventory`: `account_avatax_stock`

> Should *AvaTax* be needed for geo-localization or with the *Amazon Connector*, then install those

Reviewer: ```suggestion
Should *AvaTax* be needed for geo-localization, or with the *Amazon Connector*, then install those
```

modules individually by clicking on :guilabel:`Install` on :guilabel:`Avatax for geo localization`

> and :guilabel:`Amazon/Avatax Bridge` respectively.

Reviewer: ```suggestion
and :guilabel:`Amazon/Avatax Bridge`, respectively.
```

.. tip::

>    To install AvaTax for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil` module.

Reviewer: ```suggestion
   To install *AvaTax* for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil` module.
```

>    This action will install :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for Services`.

Reviewer: ```suggestion
   This action installs :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for Services`.
```

   * - :guilabel:`Avatax Brazil for Services`
     - `l10n_br_avatax_services`
     - This module includes the required features for tax calculation for services in the Brazil
       localization.
   * - :guilabel:`Avatax Brazil Sale for Services`
     - `l10n_br_edi_sale_services`
     - This module includes the required features for tax calculation for the sale of services in
       the Brazil localization. This includes electronic data interchange (EDI).
   * - :guilabel:`Test SOs for the Brazilian AvaTax`
     - `l10n_br_test_avatax_sale`
     - This module includes the required features for test sales orders in the Brazil localization.

Click the :guilabel:`Install` button on the module labeled, :guilabel:`Avatax`: `account_avatax`.
Doing so installs the following modules:

- :guilabel:`Avatax`: `account_avatax`
- :guilabel:`Avatax for SO`: `account_avatax_sale`
- :guilabel:`Avatax for Inventory`: `account_avatax_stock`

Should *AvaTax* be needed for geo-localization, or with the *Amazon Connector*, then install those
modules individually by clicking on :guilabel:`Install` on :guilabel:`Avatax for geo localization`
and :guilabel:`Amazon/Avatax Bridge`, respectively.

> .. tip::
>    To install *AvaTax* for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil`
>    module. This action installs :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for
>    Services`. Install the additional Brazil *Avatax* modules as needed.

Author: ```suggestion
.. tip::
   To install the *AvaTax* localization for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil`
   module. This action installs :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for
   Services`. Install the additional Brazil *Avatax* modules as needed.
   
  .. seealso::
    Reference the :doc:`Brazilian localization documentation <../../fiscal_localizations/brazil>` for more specifics on Brazilian localization. 
```

Reviewer 2: Thanks @tiku-odoo, and curious on your thoughts here @StraubCreative!

Here is the PR @vbe-odoo and I are working on, that details the l10n_br electronic invoicing & tax computation flow https://github.com/odoo/documentation/pull/8784/. 

Just my two cents– I think it is a good idea to list __all__ of the AvaTax modules that are available, even Brazil, in this main AvaTax document, but leave the local-specific install/config instructions to the fiscal localization docs, even though Brazil is the only local besides USA & Canada that Odoo supports for AvaTax.

Here is what I am suggesting, instead of the tip admonition that details Brazil AvaTax modules:
```suggestion
.. seealso::
   For localization specific *AvaTax* instructions, view the following :doc:`fiscal localization
   <../../fiscal_localizations>` documentation:

   - :doc:`../../fiscal_localizations/brazil`
```
Granted, this can be sorted out in another [IMP] PR on either this avatax.rst doc, or the brazil.rst fiscal localization doc so that we can get this important information live for 17.0+.

> fields in the :guilabel:`Taxes` section is where the *AvaTax* configurations are made and the
> credentials are entered in.

Reviewer: ```suggestion
fields in the :guilabel:`Taxes` section is where the *AvaTax* configurations are made, and the
credentials are entered.
```

> First, tick the checkbox to the left of the :guilabel:`AvaTax` settings, to activate AvaTax on the

Reviewer: ```suggestion
First, tick the checkbox to the left of the :guilabel:`AvaTax` settings, to activate *AvaTax* on the
```

> database. This is a quick, convenient way to activate and deactivate AvaTax tax calculation on the

Reviewer: ```suggestion
database. This is a quick, convenient way to activate and deactivate *AvaTax* tax calculation on the
```

Next, navigate to :menuselection:`Accounting app --> Configuration --> Accounting: Fiscal
Positions`. A :guilabel:`Fiscal Position` is listed named, :guilabel:`Automatic Tax Mapping

> (*AvaTax*)`. Click it to open *AvaTax's* fiscal position configuration page.

Reviewer 2: No italics formatting in a guilabel, it doesnt render :)
```suggestion
(AvaTax)`. Click it to open *AvaTax's* fiscal position configuration page.
```

The *AvaTax* integration is available on sale orders and invoices with the included *AvaTax* fiscal
position.

.. tip::

>    Additionally there is a :guilabel:`Tax Mapping` tab and :guilabel:`Account Mapping` tab in the

Reviewer: ```suggestion
   Additionally, there is a :guilabel:`Tax Mapping` tab and :guilabel:`Account Mapping` tab in the
```

>    :guilabel:`Automatic Tax Mapping (AvaTax)` fiscal position where mapping for products can also

Reviewer: ```suggestion
   :guilabel:`Automatic Tax Mapping (AvaTax)` fiscal position, where mapping for products can also
```
